src/js/utils/MetronomeUtil.js

```diff
@@ -91,5 +91,36 @@ module.exports = {
     });
 
     return rootTree;
+  },
+
+  parseJob(job) {
+    let {history}= job;
+
+    if (history == null) {
+
```

Comment:
So if commenting on non-eslint stylistics is considered as a waste of time and we don't do it, I wonder how we should define eslint-rules. In my opinion eslint-rules grow over time, coming out from discussions.
Defining them all at once is first incomplete and very static. :(